conf/routes

```diff
@@ -78,5 +78,9 @@ GET         /s/:supplierCodeStr              controllers.Homepage.supplierRedire
 # Paths from legacy redirects
 GET         /Voucher                         controllers.Homepage.index
 
+# Offers page or 404
+GET         /offers                          controllers.Offers.offers
+GET         /:edition/offers                 controllers.Offers.offersPage(edition: String)
```

Comment:
the "or 404" is just if you put in an invalid edition right? so we probably don't need to mention here